### DISCUSSION TITLE
Respect size attribute for form fields

### DISCUSSION THREAD

**Discussion by andypillip** [picocss/pico#457] [Ideas]
It’s usability best practice to size form fields according to the expected input. It helps users gather without reading what is expected.

Example:

```
<label>Postcode
   <input type="text" inputmode="numeric" size="6">
</label>
```

Currently, all inputs are full width.

It would be nice if the presence of a size attribute would be respected by the styles.

This could be achieved like so:

```
input:not([size]) {
  width: 100%
}
```

Checkboxes etc. are already excluded, it would suffice to add `[size]` to the list of exceptions, but I guess it would need some `display: block` as well, as the size currently is the only means used to keep the input in its own line.

**Comment by lucaslarroche** [comment]
@andypillip great idea. I like it.
Would you like to open a PR?
